integration-tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+@pytest.fixture(scope="session", autouse=True)
+def register_subman(external_candlepin, subman_session, test_config):
```

Comment:
I still would like consider the option to use auto use at once for all tests. In my understanding multiple files would need to run the tests on a registered system that will cause unnecessary registration/unregistration. and most importantly a test like test_version or man page tests would not suffer from a system being registered already. And a big positive would be eliminating multiple subman register/unregister process. Your thoughts??

Reply:
Ah, that should not be an issue: since your new fixture has a "session" scope, even if it is used in multiple places it is instantiated only once. So having the marker I suggested in multiple files will not have a negative side effect.